docs/dxp/7.x/en/site-building/developer-guide/developing-themes/developing-themelets.md

```diff
@@ -1,6 +1,8 @@
 # Developing Themelets
 
-Themelets are Small, extendable, and reusable pieces of code containing CSS and JavaScript, that you can extend Themes with. They can be shared with other developers to provide common components for Themes. The steps below show how to use the Liferay Theme Generator's Themelets sub-generator to create a Themelet.
+Themelets are small, extendable, and reusable pieces of code containing CSS and JavaScript, that you can extend Themes with. They can be shared with other developers to provide common components for Themes. The steps below show how to use the Liferay Theme Generator's Themelets sub-generator to create a Themelet.
+
+<!-- screenshot of ... end result is that even possible in this case? -->
```

Comment:
I have the end result screenshots after the last step in deploying the example. Before that these are just small modifications to the existing Theme, and may not make sense out of context.